articles/communication-services/quickstarts/rooms/includes/rooms-quickstart-javascript.md

```diff
@@ -90,14 +89,14 @@ console.log(`Retrieved Room with ID ${roomId}`);
 
 ### Update the lifetime of a room
 
-The lifetime of a `room` can be modified by issuing an update request for the `ValidFrom` and `ValidUntil` parameters. A room can be valid for a maximum of six months. 
+The lifetime of a `room` can be modified by issuing an update request for the `ValidFrom` and `ValidUntil` parameters. A room can be valid for a maximum of six months.
 
 ```javascript
 validFrom.setTime(validUntil.getTime());
 validUntil.setTime(validFrom.getTime() + 5 * 60 * 1000);
 
 // request payload to update a room
-const updateRoomRequest = {
+const updateRoomOptions = {
 validFrom: validFrom,
 validUntil: validUntil,
 roomJoinPolicy: "CommunicationServiceUsers",
```

Comment:
Lets not add new participants to the update call

Reply:
We have a separate endpoints for adding participants. Lets not suggest adding them through updateRoom call